src/services/logger.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response } from 'express';
+import * as winston from 'winston';
+
+export class LoggerService {
+  public logger;
+  constructor(){
+    this.logger = winston.createLogger({
+      transports: [
+        new winston.transports.Console()
+       ]
+    });
+  }
+
+  logMethodInfo (req: Request, res: Response, next): void {
```

Comment:
I believe that single responsibility of LoggerService is to log/error/debug/warn some data or errors. But is should know nothing about request, response or even next calback